BossMod/Util/UIMisc.cs

```diff
@@ -15,5 +15,17 @@ public static bool DangerousButton(string label)
                 ImGui.SetTooltip("Hold shift");
             return res;
         }
+
+        public static void TextUnderlined(System.Numerics.Vector4 colour, string text)
```

Comment:
Any reason to pass color as vector4 here rather than uint? The rest of the plugin uses uint colors everywhere

Reply:
It was purely because I default to using dalamud colours and didn't notice you don't. I can make a UIColour class or a converter, whichever you'd rather have

Reply:
Oh, also because all Imgui takes in vector4 for text colours